utilities/tasks/addMissingPostsToUser/addMissingPostsToUser.js

```diff
@@ -0,0 +1,52 @@
+const _ = require('lodash');
+const { postsUtil } = require('utilities/steemApi');
+const { checkAppBlacklistValidity } = require('utilities/helpers').appHelper;
+const Sentry = require('@sentry/node');
+const { sendSentryNotification } = require('utilities/helpers/sentryHelper');
+const UserModel = require('models/UserModel');
+const PostModel = require('models/PostModel');
+const { reblogPostParser } = require('parsers/userParsers');
+const { parseBodyWobjects } = require('../../helpers/postHelper');
+
+module.exports = async (name) => {
+  const { user } = await UserModel.checkAndCreate(name);
+  user.wobjects_weight = 25000;
+
+  const { posts, err } = await postsUtil.getPostsByAuthor(name);
+  if (err || !posts) {
+    sendSentryNotification();
+    Sentry.captureException(err);
+  }
+
+  const reblogPosts = _.reject(posts, { author: name });
+  reblogPosts.forEach((post) => parseAndSavePost(post, post.author));
```

Comment:
are you sure they will be saved until your main loop?
you return call of async function inside sync,
use for of with await instead